components/proxy/src/main/java/com/hotels/styx/routing/config/StyxObjectDefinition.java

```diff
@@ -67,7 +68,7 @@ public JsonNode config() {
     }
 
     static class Builder {
-        private List<String> tags = ImmutableList.of();
+        private List<String> tags = emptyList();
```

Comment:
We could, but I think that `emptyList()` often reads better as it communicates exactly its intent, and I generally slightly favour JDK methods over library methods unless there's a reason not to. It's a fine point, I know. I don't feel the same about `singletonList()` though (not least because there isn't a `singletonSet()`).